app/models/like.rb

```diff
@@ -0,0 +1,12 @@
+class Like < ApplicationRecord
+  belongs_to :post
+  belongs_to :author, class_name: 'User'
+
+  after_save :update_likes_counter
+
+  private
+
+  def update_likes_counter
+    post.update(likes_counter: post.likes.count)
+  end
```

Comment:
- [ ] Great job using [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) in your models. 👍